File: gfg/python/leader_in_array.py
def printLeaders(elements,size):
    for i in range(0,size):
        for j in range(i+1,size):
            if elements[i] <= elements[j]:
                break

        if j == (size - 1):
            print(elements[i])

# Driver function
arr=[16, 17, 4, 3, 5, 2]
printLeaders(arr, len(arr))

File: gfg/js/leader_in_array.js
function printLeaders(elements,size){
    for(let i=0;i<size;i++){
        let j;
        for(j=i+1;j<size;j++){
            if(elements[i] <= elements[j]){
                break
            }
        }

        if (j == size){
            console.log(elements[i])
        }
    
    }
}

const elements = [16, 17, 4, 3, 5, 2];
const n = elements.length;
printLeaders(elements, n)

